File: tests/fixtures/go/sample.go
package sample

// Helper doubles an int.
func Helper(x int) int {
	return x * 2
}

// TopLevelOne calls Helper.
func TopLevelOne(x int) int {
	return Helper(x)
}

// Greeter is a struct.
type Greeter struct {
	Name string
}

// Greet is a method on Greeter that calls Helper.
func (g *Greeter) Greet() int {
	return Helper(len(g.Name))
}

File: tests/fixtures/java/Sample.java
package sample;

public class Sample {
    public static int helper(int x) {
        return x * 2;
    }

    public static int topLevelOne(int x) {
        return helper(x);
    }
}

class Greeter {
    private String name;

    public Greeter(String name) {
        this.name = name;
    }

    public int greet() {
        return Sample.helper(this.name.length());
    }
}
